config.schema.json

```diff
@@ -37,7 +37,6 @@
             "dockContactSensor": {
                 "type": "boolean",
                 "title": "Show dock status as a contact sensor",
-                "default": true,
```

Comment:
Why is this removed?

Reply:
I had removed it to be consistent across all of the added sensors, when doing the docs. After looking back through the history I decided to put it back and must not have committed it.